src/expandableCalendar/WeekCalendar/index.tsx

```diff
@@ -52,7 +54,10 @@ const WeekCalendar = (props: WeekCalendarProps) => {
 
   useDidUpdate(() => {
     if (updateSource !== UpdateSources.WEEK_SCROLL) {
-      const pageIndex = items.current.findIndex(item => sameWeek(item, date, firstDay));
+      const pageIndex = items.current.findIndex(
+        item => (numberOfDays && numberOfDays > 1) ?
+          sameWeekRange(item, date, numberOfDays, item) :
```

Comment:
I should name this better, I pass the date for the first day in the custom week (for example in a 3-day view it would be a 3 days week where the current date is the first day in one of those weeks) because those weeks have custom length we can't rely on any fixed first day of the week

Reply:
Yes, probably a renaming issue - see my comment above